### PR TITLE
beta -> RC

### PR DESCRIPTION
**Pull request by birkskyum**
From Beta
Screenshot 2024-03-23 at 13 56 49

To RC 🎉
Screenshot 2024-03-23 at 14 31 35

### CODE REVIEW

File: docs/routes/getting-started/what-is-solidstart.md [hunk 1]
```diff
@@ -8,8 +8,7 @@ active: true
 # What is SolidStart?
 
 <aside title="Beware of dragons" type="warning">
-  We are very excited about SolidStart, but it is currently in release candidate. There will very likely be <b>bugs and
-  missing documentation</b>. Come at it with a growth attitude, have fun, and don't use it yet for
+  We are very excited about SolidStart, but it is currently in release candidate. There will very likely be <b>bugs and missing documentation</b>. Come at it with a growth attitude, have fun, and don't use it yet for
```
Comment (atk) [inline docs/routes/getting-started/what-is-solidstart.md:11]: Strange wording "in release candidate", maybe better use "a release candidate".

Comment (birkskyum) [reply docs/routes/getting-started/what-is-solidstart.md]: Good point. I added "stage" - does that work?

File: README.md [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
 
-### Start has just entered a new Beta Phase
+### StartStart has just entered a new Release Candidate Phase
 
 v0.4.x marks a significant change in the project. Please check the updated docs and example projects to see how things have changed. A summary of the changes can be found in the [RFC](https://github.com/solidjs/@solidjs/start/discussions/1052).
 
 ------------------------------------
 
-This is the home of the Solid app framework. This is still a **work in progress**. Many features are missing or incomplete. Experimental status does not even mean beta status. Patch releases will break everything.
+This is the home of the StartStart, the Solid app framework.
```
Comment (lxsmnsyc) [inline README.md:13]: Double "Start", same as L7

**Comment by ryansolid** [comment]
Thanks

**Pull request merged**